src/test/java/core/basesyntax/service/FileServiceTest.java

```diff
@@ -0,0 +1,112 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.service.impl.FileServiceImpl;
+import org.junit.Test;
+
+public class FileServiceTest {
+    
+    private static final String FIRST_FILE_PATH = "src/test/resources/defaultFile";
+    private static final String SECOND_FILE_PATH = "src/test/resources/readFrom";
+    private static final String EMPTY_FILE_FROM_NAME = "src/test/resources/emptyFile";
+    private static final String NONEXISTENT_FILE_NAME = "nonexistent";
+    private static final String FILE_TO_WRITE_NAME = "src/test/resources/fileTo";
+    
+    @Test
+    public void fileService_readFromFile_firstCase_Ok() {
```

Comment:
fileService - class name
```suggestion
    public void readFromFile_firstCase_Ok() {
```